recipes-graphics/libplanes/libplanes/0001-Use-python3-by-default.patch

```diff
@@ -0,0 +1,54 @@
+From ba4cc0d7b08efecd2b2bfb04c7a70187d7056304 Mon Sep 17 00:00:00 2001
```

Comment:
I feel that this patch would better go directly to https://github.com/linux4sam/libplanes . What do you think?

Reply:
already done https://github.com/linux4sam/libplanes/pull/3

Reply:
Thanks! I can merge the first commit while the pull request on libplanes gets accepted...

Reply:
well, we need the second commit for it to work with master OE-Core/yocto since py2 is being thrown out next week. So I would suggest taking both. If libplane commit requires changes after review, I will update the patch here as well accordingly.

Reply:
We need to make more tests with python3 and see if everything is fine, so I will postpone your pull request until then.